mono/mini/tramp-amd64.c

```diff
@@ -29,6 +29,10 @@
 #include "mini-amd64.h"
 #include "debugger-agent.h"
 
+#ifdef ENABLE_INTERPRETER
+#include "interpreter/interp.h"
+#endif
+
```

Comment:
because `mono_arch_get_enter_icall_trampoline` is defined in `interp.h`, but implemented in `tramp-$arch.c`.  maybe I should move that definition to somewhere else? `mini.h`?